guix/README.org

```diff
@@ -100,20 +100,25 @@ That gets built by =src/build/build-version.scm= so let us replace that.
 #+end_src
 
 #+begin_src scheme :tangle gerbil-build-utils.scm
-  (include "gerbil-stamp.scm")
-
-  (define gerbil-build-version
-    `(with-output-to-file "src/build/build-version.scm"
-       (lambda ()
-         (write
-  	'(let* ((gerbil-version-path
-  		 (path-expand "gerbil/runtime/version.ss" (getenv "GERBIL_SOURCE")))
-  		(gerbil-version-text
-  		 (string-append "(def (gerbil-version-string) \"" ,gerbil-stamp-version "\")\n")))
-  	   (display "... write ") (display gerbil-version-path) (newline)
-  	   (call-with-output-file `(path: ,gerbil-version-path create: maybe append: #f truncate: #t)
-  	     (lambda (port) (display gerbil-version-text port))))))))
-  
+    (include "gerbil-stamp.scm")
+
+    (define gerbil-make-gambit
+     `(substitute* "src/build.sh"
+      (("\\s+\\(cd gambit && make -j \\$\\{GERBIL_BUILD_CORES:-1\\} core\\) \\|\\| die")
+       "   (cd gambit && make -j ${GERBIL_BUILD_CORES:-1} bootstrap) || die\n   (cd gambit && make -j ${GERBIL_BUILD_CORES:-1} from-scratch) || die")))
+
+    (define gerbil-build-version
+      `(with-output-to-file "src/build/build-version.scm"
+         (lambda ()
+           (write
+    	'(let* ((gerbil-version-path
+    		 (path-expand "gerbil/runtime/version.ss" (getenv "GERBIL_SOURCE")))
+    		(gerbil-version-text
+    		 (string-append "(def (gerbil-version-string) \"" ,gerbil-stamp-version "\")\n")))
+    	   (display "... write ") (display gerbil-version-path) (newline)
+    	   (call-with-output-file `(path: ,gerbil-version-path create: maybe append: #f truncate: #t)
+    	     (lambda (port) (display gerbil-version-text port))))))))
```

Comment:
Go away post-#1074

Reply:
> Go away post-#1074

I do not see a MANIFEST file in every single commit, nor is there one in my test builds. Am I missing something? How does that file get created if I am building from HEAD without git?

Reply:
There will be a MANIFEST file on git iff the commit is a release.
On the other hand, you can have Guix create its own MANIFEST file to build without git.
That's how my upcoming Nix build will work: no more fancy patches, just create a MANIFEST.
Create your own local one with ./manifest.sh.